views-vues/assurance-level-requirement/en/assurance-level-requirement.html

```diff
@@ -0,0 +1,250 @@
+---
+layout: default
+title: Assurance Level Requirement (draft)
+lang: en
+altLang: fr
+altLangPage: nuage-outil-decisionnel
+collectionDirectory: views-vues/assurance-level-requirement
+version: Assessment Version 1.0
+questions:
+  "1": Inconvenience, distress, loss of standing or reputation
+  "1-a1": An inconvenience, distress or damage to the standing or reputation of any party
+  "1-a2": A serious short-term or a limited long-term inconvenience, distress or damage to the standing or reputation of any party
+  "1-a3": A serious long-term inconvenience, distress or damage to the standing or reputation of any party
+  "1-a4": A severe and permanent inconvenience, distress or damageto the standing or reputation of any party
+  "1-hint-a1":
+    • Alternatives are available with little or no delay and no additional costs or degradation of service quality &#10;
```

Comment:
I'd replace these Word style markers with regular `-`

Reply:
dashes will make the items into an array under the heading, when they need to be read as a text block. 

It would be possible to use that and have it iterate over the array every time they are needed to print, but that seems the less useful option, but I'll listen if you think that would still be preferred.

Reply:
Ah, gotcha. would the `<ul>`/`<li>` formatting in some of the other sections make sense then?

Reply:
it *might* be possible, but would be more trouble than its worth.

The input is used in the questionnaire as a more information section
<img width="859" alt="Screen Shot 2020-09-21 at 12 40 52 PM" src="https://user-images.githubusercontent.com/55841241/93796459-dfa08b80-fc08-11ea-9f6b-8c2a5e7217c7.png">

The questionnaire is made up of a list which is turned into input using data-wb-fieldflow: 
```
<li data-wb-fieldflow='{ "action": "removeClass", "source": "#table-question-{{ question }}-level-{{ level }}, #table-question-{{ question }}-assessment-{{ level }}, #table-question-{{ question }}-harm-{{ level }}", "class": "hidden" }'>
   {{ page.questions[ questionAnswerString ] }}
   &lt;abbr title=&quot;{{ page.questions[ questionHintString ] | prepend: " " }}&quot;&gt;({{ page.moreInfo }})&lt;/abbr&gt;
</li>
```

In order to add custom HTML into the actual questions, you have to escape it, as I did above.
I previously tried to implement it using lists, but I never found a way for it to be escaped for the list, still be properly read by liquid, and have it render on the other side.

Reply:
Sorry, I'm not super familiar with the way this was setup originally, but using `title` as the only way to convey the information wouldn't meet accessibility requirements and doesn't work on mobile either

Reply:
I've changed it so that now it injects a button to an overlay from WET, this also allowed me to change over to a `<ul>` format.

<img width="1278" alt="Screen Shot 2020-09-21 at 2 38 46 PM" src="https://user-images.githubusercontent.com/55841241/93807311-7163c500-fc18-11ea-88e1-c46daaa1eddc.png">